mujoco_warp/_src/passive.py

```diff
@@ -466,16 +465,15 @@ def _flex_bending(
 @event_scope
 def passive(m: Model, d: Data):
   """Adds all passive forces."""
+  d.qfrc_spring.zero_()
```

Comment:
In #355 we learned that zeroing arrays does show up in benchmarks and is not insignificant!

Can you think of a way to structure this code so that we don't need to zero out all the arrays at every step?

Reply:
restructured the code to avoid zeroing arrays (except `qfrc_gravcomp`) at each step